internal/crypto/ptls/default_test.go

```diff
@@ -0,0 +1,318 @@
+// Copyright 2021-2024 the Pinniped contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package ptls
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"slices"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestTLSSecureCipherSuites checks whether golang has changed the list of secure ciphers.
```

Comment:
This nice test could use a comment explaining why it exists and what we should reconsider about our code if it ever fails.

Reply:
done with next push